src/main/java/slimeknights/mantle/command/MantleCommand.java

```diff
@@ -33,8 +33,6 @@ public class MantleCommand {
   /** Suggestion provider that lists tags values for this registry */
   public static SuggestionProvider<CommandSourceStack> REGISTRY_VALUES;
   /** Suggestion provider that lists registered book ids **/
-  public static SuggestionProvider<CommandSourceStack> REGISTERED_BOOKS;
```

Comment:
Same as above comment, fine to remove this for 1.20. Will want to keep it around for 1.19 to prevent a breaking change.